app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+  <h1 class="text-center">Public Recipes</h1>
+  <% @public_recipes.each do |public_recipe| %>
+    <table class="table table-borderless border w-75 m-auto">
+      <tbody>
+        <tr scope="col">
+          <td><%= public_recipe.name %></td>
+          <td>Total food items: <%= public_recipe.food.count %></td>
+        </tr>
+        <tr scope="col">          
+          <td>By <%= public_recipe.user.name %></td>
+          <td>Total price: $<%= public_recipe.food.sum(:price) %></td>
+        </tr>
+      </tbody>
```

Comment:
- [ ] According to the project requirement, clicking on a recipe in the public recipe list should take the user to the recipe details. Kindly implement this, please. This is so that users can view the food ingredients available in the recipe.
![image](https://user-images.githubusercontent.com/94551484/221008832-f27f2557-d40f-416c-8cf9-7a4588b9f2b7.png)